eclipsePlugin-junit/build.gradle

```diff
@@ -0,0 +1,7 @@
+apply from: "$rootDir/gradle/checkstyle.gradle"
+
+dependencies {
+  compile project(':eclipsePlugin')
+  testCompile 'junit:junit:4.12'
+  testCompile 'org.mockito:mockito-core:2.7.22'
+}
```

Comment:
Yep, with the TODO that the new project should be merged to the old one as soon as we have working build/test cycle again.